pkg-manager/core/src/parseWantedDependencies.ts

```diff
@@ -1,7 +1,6 @@
 import { parseWantedDependency } from '@pnpm/parse-wanted-dependency'
 import { type Dependencies } from '@pnpm/types'
-import { whichVersionIsPinned } from '@pnpm/which-version-is-pinned'
-import { type PinnedVersion, type WantedDependency } from '@pnpm/resolve-dependencies/lib/getWantedDependencies'
+import { type WantedDependency } from '@pnpm/resolve-dependencies/lib/getWantedDependencies'
```

Comment:
Shouldn't `WantedDependency` be reexported at `index.ts` to enable this?

```suggestion
import { type WantedDependency } from '@pnpm/resolve-dependencies'
```